lib/components/StatusTag/StatusTag.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC } from "react";
+
+import Icon from "@mdi/react";
+import {
+  mdiAlert,
+  mdiAlertCircle,
+  mdiMinusCircle,
+  mdiCheckCircle,
+} from "@mdi/js";
+
+import { StatusTagProps } from "./StatusTag.types";
+import {
+  IconWrapper,
+  StatusTagWrapper,
+  StyledOverlineLarge,
+} from "./StatusTag.styled";
+
+const StatusTag: FC<StatusTagProps> = ({
+  label,
+  size = "medium",
+  variant = "neutral",
+}) => {
+  const getIcon = () => {
+    switch (variant) {
+      case "error":
+        return mdiAlert;
+      case "warning":
+        return mdiAlertCircle;
+      case "success":
+        return mdiCheckCircle;
+      case "neutral":
+        return mdiMinusCircle;
+      default:
+        return mdiMinusCircle;
+    }
+  };
```

Comment:
```suggestion
const StatusTag: FC<StatusTagProps> = ({
  label,
  size = "medium",
  variant = "neutral",
  icon
}) => {
```

please make the icon passable as a prop, so we can use other icons or images here. when necessary.
you can make it a react node type so anything can be passed.

Reply:
it would be wise to allow/account for the icon being also nullable, in case we need to render no icons there.

Reply:
I will do it, no problem. I just thought that it was important to "enforce" the use of the right icon, according to the Figma specification (and also because the task mentioned only size and variant as params, not the icon). But I will work on these changes then.

Reply:
Done!

Reply:
Shouldn't we provide a customizable component and a collection of components as required by design, so when using the Status Tag Componente we could be more brief like: 
```
<Status satisfatory/>
<Status high-risk/>
```